server/services/personalPageService.ts

```diff
@@ -313,7 +325,21 @@ export default class PersonalPageService {
     return { personId, addresses }
   }
 
-  private physicalCharacteristics(prisonerData: Prisoner, inmateDetail: InmateDetail): PhysicalCharacteristics {
+  private physicalCharacteristics(
+    prisonerData: Prisoner,
+    inmateDetail: InmateDetail,
+    prisonPerson: PrisonPerson,
+  ): PhysicalCharacteristics {
+    let height = prisonerData.heightCentimetres
+      ? `${(prisonerData.heightCentimetres / 100).toString()}m`
+      : 'Not entered'
+    let weight = prisonerData.weightKilograms ? `${prisonerData.weightKilograms}kg` : 'Not entered'
+
+    if (prisonPerson) {
+      height = prisonPerson.physicalAttributes.height.toString()
+      weight = prisonPerson.physicalAttributes.weight.toString()
+    }
+
```

Comment:
Display logic in services.... although looks like that is how the rest of the file works